src/App.jsx

```diff
@@ -14,33 +17,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_NAME = 'name';
+const SORT_BY_LENGTH = 'length';
+const REVERSE = 'reverse';
+let isSortParametersNull;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export function App() {
+  const goodsCopy = [...goodsFromServer];
+  const [sortType, setSortType] = useState(null);
+  const [reverseOfArray, setReverseOfArray] = useState(null);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortType === SORT_BY_NAME) {
+    goodsCopy.sort((goods1, goods2) => goods1.localeCompare(goods2));
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (sortType === SORT_BY_LENGTH) {
+    goodsCopy.sort((goods1, goods2) => goods1.length - goods2.length);
+  }
+
+  if (reverseOfArray === REVERSE) {
+    goodsCopy.reverse();
+  }
+
+  if (sortType === null && reverseOfArray === null) {
+    isSortParametersNull = null;
+  } else {
+    isSortParametersNull = 'yes';
+  }
```

Comment:
You don't this logic, however a few suggestions for the future:
- You don't need to compare with null/undefined/empty string and other falsy values, you can just check if the value exists https://developer.mozilla.org/en-US/docs/Glossary/Falsy
- `yes`/`null` confusing values for a variable in this case, it could be just boolean